lib/internal/modules/esm/get_format.js

```diff
@@ -41,15 +42,37 @@ function getDataProtocolModuleFormat(parsed) {
   return mimeToFormat(mime);
 }
 
+const DOT_CODE = 46;
+const SLASH_CODE = 47;
+
+/**
+ * Returns the file extension from a file: URL. Should give similar result than
```

Comment:
nit

```suggestion
 * Returns the file extension from a file:// URL. Should give similar result than
```

Reply:
That's a good catch, that made me wonder if there was an "official" correct naming for these, https://www.rfc-editor.org/rfc/rfc8089.html is using `"file" URI`, however I haven't seen this naming much, imo ``a `file:` URL`` is clearer. Anyway, in this case the protocol has no importance, we are only parsing the pathname so it would work with any URL, so let's simplify that comment.
```suggestion
 * Returns the file extension from a URL. Should give similar result to
```